src/GLONASS_IFB_CORRECTION/glonass_IFB_correction.h

```diff
@@ -0,0 +1,71 @@
+/* 
+ *   GLONASS INTER-FREQUENCY PHASE BIASES (IFB) CORRECTION MODULE
+ */
+
+#ifndef GLONASS_IFB_CORRECTION_H
+#define GLONASS_IFB_CORRECTION_H
+
+#include "rtklib.h"
+
+/* -------------------------------------------------------------------------------------------------------------- */
+/* model paramters */
+
+#define GLO_IFB_MIN_GLO_DT_SEARCH  -0.3   /* min glo_dt value while searching (cycles/frequency_number) */
```

Comment:
It seems like most of these defines are only used in glonass_IFB_correction.c. So move it there.